Index page with "icons"

doekman opened the pull request:
As proposed in issue #25.

doekman committed:
Index page with icons (issue #25)

doekman committed:
README updated

doekman committed:
Updated README

doekman committed:
Fixed casing in README

rithvikvibhu commented:
@doekman the page seems to be broken. Take a look?
https://htmlpreview.github.io/?https://github.com/doekman/awfice/blob/index/index.html
![image](https://user-images.githubusercontent.com/5113343/98457989-14b16f00-21b2-11eb-852b-5b0959293b12.png)

doekman committed:
Fixed ambiguous ampersands

doekman commented:
That looks like a work of art 😉!

I think this is an issues with _htmlpreview.github.io_: it's a client-side solution. It uses [document.open/write/close](https://github.com/htmlpreview/htmlpreview.github.com/blob/master/htmlpreview.js#L66) to render the document. The HTML-source of `index.html` that loads fine from a local webserver, gets mangled up by this method.

Another solution would be to use [GitHub Pages](https://pages.github.com). They could be served from folder within the repo... Something like: `https://awfice.github.io`.

rithvikvibhu commented:
Ah, I see. Yeah, GH Pages seems to be simple and the easiest. @zserge is the only one who can enable it I think.

zserge commented:
Thanks, enabled github pages for this repo, now some "apps" are accessible:

* https://zserge.com/awfice/draw.html 
* https://zserge.com/awfice/edit.html
* https://zserge.com/awfice/code.html
* https://zserge.com/awfice/calc.html
* https://zserge.com/awfice/beam.html

doekman committed:
Merge branch 'main' into index

doekman committed:
Move away from htmlpreview.github.io

doekman commented:
I've synced upstream (calculator.html was added), so this PR will merge again.

Also, I've replaced the preview URL from `htmlpreview.github.io` to something configurable, which defaults to `https://zserge.com/awfice` (see `.ok` file for information). This way, one can test this branch before deploying.

I'm not too sure how this repository is hosted via GitHub Pages over there, since the `README` here, is called `index` over there. So I renamed the startpage to `start.html` (which is a better name anyways). Hope this works.
